aeon/transformations/collection/channel_selection.py

```diff
@@ -289,15 +289,8 @@ class pair across each channel.
     """
 
     _tags = {
-        "scitype:instancewise": True,  # is this an instance-wise transform?
-        "univariate-only": False,  # can the transformer handle multivariate X?
-        "X_inner_mtype": "numpy3D",  # which mtypes do _fit/_predict support for X?
-        "y_inner_mtype": "numpy1D",  # which mtypes do _fit/_predict support for y?
-        "requires_y": True,  # does y need to be passed in fit?
-        "fit_is_empty": False,  # is fit empty and can be skipped? Yes = True
-        "skip-inverse-transform": True,  # is inverse-transform skipped when called?
-        "capability:unequal_length": False,
-        # can the transformer handle unequal length time series (if passed Panel)?
+        "capability:multivariate": True,
+        "skip-inverse-transform": True,
```

Comment:
```suggestion
        "capability:multivariate": True,
        "skip-inverse-transform": True,
        "y_inner_mtype": "numpy1D",
        "requires_y": True,
```

Im surprised theres no test for `requires_y`